README.md

```diff
@@ -31,6 +31,7 @@ samples, guidance on mobile development, and a full API reference.
 4. All the code that we have to work with is present in the lib folder.
 5. The screens folder contains the basic layout of each individual page. We add more widgets and make up the UI of the page.
 6. The widgets folder has individual widgets which can be used on multiple pages or are large in size and would reduce the readabilty and make a single page's code too long. 
+7. Before making a pull request, format the code with `Ctrl+Alt+L`. Add a comma after all the closing brackets as it will help the IDE in aligning the brackets and formating the code. For more on this checkout [Code Formatting](https://flutter.dev/docs/development/tools/formatting) .
```

Comment:
typo in formatting(written as formating)

Reply:
sent a PR fixing it